### PR TITLE
test: move e2e tests to cypress remove protractor and add stub for ci workflow

### PR DESCRIPTION
**Pull request by roedoejet**
very basic stub implementation of cypress e2e tests that works on CI

### CODE REVIEW

**Comment by joanise** [comment]
Humph, you reformated the spacing in package.json and package-lock.json, making this harder to review, and probably the cause of the conflict.

**Comment by joanise** [comment]
> you reformated

you, or more likely pre-commit, I know...

**Comment by joanise** [comment]
codecov.io support cypress coverage, it would be lovely to integrate.

**Comment by joanise** [comment]
Potential problem: if I launch `npm run cypress:open` and then click on Component Testing, I get this error:
`Error: error TS5012: Cannot read file 'C:/Users/joanise/sandboxes/wordweaver-UI/projects/word-weaver/projects/word-weaver/tsconfig.app.json': ENOENT: no such file or directory, open 'C:/Users/joanise/sandboxes/wordweaver-UI/projects/word-weaver/projects/word-weaver/tsconfig.app.json'.`
Notice how the path repeats `projects/word-weaver` twice.

File: package.json [hunk 1]
```diff
@@ -1,159 +1,163 @@
...
+        "@fortawesome/free-brands-svg-icons": "^5.12.1",
+        "@fortawesome/free-solid-svg-icons": "^5.12.1",
+        "@ngrx/effects": "^14.3.3",
+        "@ngrx/entity": "^14.3.3",
+        "@ngrx/router-store": "^14.3.3",
+        "@ngrx/store": "^14.3.3",
+        "@ngrx/store-devtools": "^14.3.3",
+        "@ngx-translate/core": "12.0.0",
+        "@ngx-translate/http-loader": "4.0.0",
+        "@types/cypress": "^1.1.3",
```
Comment (joanise) [inline package.json]: `@types/cypress` triggers a deprecation warning and is not required anyway, tests run without it.
```suggestion
```

File: projects/word-weaver/src/environments/environment.prod.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export const environment = {
   },
   production: true,
   test: false,
-  base: `https://test.kawennonnis.ca`,
+  base: `https://roedoejet.github.io/wordweaver-ui`,
```
Comment (joanise) [inline projects/word-weaver/src/environments/environment.prod.ts:17]: Ah, I guess this is where you want to deploy our default demo one. OK

**Review by joanise — APPROVED** [review]
I wish whole file reformatting wasn't so automatic in your workflow, but otherwise as a first step to get an e2e framework going this is good. See other comments for suggested improvements, but which are not required for this PR.

**Comment by roedoejet** [comment]
> codecov.io support cypress coverage, it would be lovely to integrate.

Agreed. I think this is a bit more involved though. I'll add it to clickup.

**Pull request merged**